tools/macos_buildenv.sh

```diff
@@ -35,16 +35,13 @@ case "$1" in
         ;;
 
     setup)
-        # Minimum required by Qt 5.12
-        MACOSX_DEPLOYMENT_TARGET=10.12
```

Comment:
This is still required. Please revert.

Reply:
This was never exported, a good evidence that it was not required. I think this is because we do not use framework functions directly. I have added https://github.com/mixxxdj/mixxx/pull/4225/commits/b68d3b985c804390b0adbea509f1ce38cef73d9a
to export it just in case. Hopefully everything still works.

Reply:
The build will still work fine without this. But the executable probably won't run on as old of macOS version as it could.